### ISSUE TITLE
Ads showing on lmgtfy.com

### ISSUE THREAD

**Issue by ryanbr** [uBlockOrigin/uAssets#7678]
### URL(s) where the issue occurs

`https://lmgtfy.com/?qtype=search&t=w&segment=lmgtfy2&is_ad=0&gclid=&q=test&as=0&engine=&media_type=`

Shows ads in uBO (but not in ABP)

### Describe the issue

The following elements aren't being applied `##.ads-bing-bottom`  and `##.ads-bing-top`

### Screenshot(s)

![lgtfy-top](https://user-images.githubusercontent.com/1659004/87836862-b7816d00-c8e5-11ea-9a2c-1b941461ec90.png)
![lgtfy-bottom](https://user-images.githubusercontent.com/1659004/87836873-c10ad500-c8e5-11ea-9d27-5a4fc8e01267.png)

### Versions

- Browser/version: Brave 1.11.97
- uBlock Origin version: 1.27.10

### Settings

-

### Notes

Issue doesn't occur in ABP 3.9.1

**Comment by gwarser** [comment]
@gorhill cosmetic filters does not work in `srcdoc` iframes. 


Workaround `lmgtfy.com##[data-s1search$="p"],[data-s1search$="m"]` https://www.reddit.com/r/uBlockOrigin/comments/ho64xq/let_me_google_that_for_you_ads_not_blocked/

**Comment by ryanbr** [comment]
https://github.com/easylist/easylist/commit/a244d5d

A work around

**Comment by gorhill** [comment]
I don't think the browser allows content scripts to execute in `srcdoc` frames, so the only way to fix this is filter-based.

**Comment by gwarser** [comment]
> content scripts

You think about generic filters? DOM surveyor? Specific-generic and completely specific filers also does not work. Generic filters from EasyList work in ABP.

**Comment by gorhill** [comment]
> Specific-generic and completely specific filers also does not work.

Isn't the fix here to use a specific cosmetic filter?

**Comment by gwarser** [comment]
> Isn't [the fix here](https://github.com/uBlockOrigin/uAssets/commit/65dc035cba030a2dbcc58ba802019511c059f822) to use a specific cosmetic filter?

No. This hides complete iframe instead of element in its content.

But I also see that `insertCSS` with `allFrames: true` also does not work.

**Comment by gorhill** [comment]
`match_about_blank` does not seem to make a difference either. Puzzling.

**Comment by gorhill** [comment]
Strike, I hadn't try changing the calls to `insertCSS` in cosmetic filtering engine -- it does work when using `matchAboutBlank` set to `true` in calls to `insertCSS` in <https://github.com/gorhill/uBlock/blob/master/src/js/cosmetic-filtering.js>.
